### PR TITLE
Add deprecation and future disabling of playstore builds warning to banner

### PR DESCRIPTION
**Pull request by agnostic-apollo**

### CODE REVIEW

**Comment by agnostic-apollo** [comment]
Working as desired on playstore `v0.101` and Github debug build `v0.117`. No banner with the later. If any changes need to be made, let me know. An additional newline at the end could be added.

playstore-banner

**Comment by Grimler91** [comment]
I think it could be a bit shorter and rather link to places with more info (to increase the chances of people reading it). Suggestion (with slightly higher column width but not wider than your current resolution):

```
You are using a very old version of Termux, probably
installed from Google Play Store.  The termux apps will
be removed from Play store in the near future.  Instead
you are encouraged to install from F-Droid or Github
Debug builds (see [1]).  You can backup all your
current data before changing installation source, and
then restore it afterwards, by following instructions
in the wiki [2].  Check the changelog [3] for all the
new features and fixes that you are currently missing.

[1] https://github.com/termux/termux-app#installation
[2] https://wiki.termux.com/wiki/Backing_up_Termux
[3] https://github.com/termux/termux-app/releases

```

File: packages/termux-tools/login [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 if [ $# = 0 ] && [ -f @TERMUX_PREFIX@/etc/motd ] && [ ! -f ~/.hushlogin ] && [ -z "$TERMUX_HUSHLOGIN" ]; then
 	cat @TERMUX_PREFIX@/etc/motd
+
+	# TERMUX_VERSION env variable has been exported since v0.107 and PATH was being set to following value in <0.104. Last playstore version was v0.101.
+	if [ -f @TERMUX_PREFIX@/etc/motd-playstore ] && [ -z "$TERMUX_VERSION" ] && [ "$PATH" = "@TERMUX_PREFIX@/bin:@TERMUX_PREFIX@/bin/applets" ]; then
+		printf '\033[0;31m'; cat @TERMUX_PREFIX@/etc/motd-playstore; printf '\033[0m'
+	fi
```
Comment (Grimler91) [inline packages/termux-tools/login:9]: Should probably be outside `if [ ! -f ~/.hushlogin ] && [ -z "$TERMUX_HUSHLOGIN" ]`, we want to force people to see the message and really encourage them to move to a more up to date version (even if they have previously silenced the motd)

Comment (agnostic-apollo) [reply packages/termux-tools/login:9]: Yeah that makes sense. I will do it.

**Comment by agnostic-apollo** [comment]
>I think it could be a bit shorter and rather link to places with more info (to increase the chances of people reading it).

I was thinking it was a bit lengthy too, but had to provide more info on why moving should be a must now and that they won't lose data during move. Maybe we can make a short wiki page for it too and possibly provide instructions to separately backup home and usr so that they can be done one at a time for people with low storage, possibly with `xz` compression.

> Suggestion (with slightly higher column width but not wider than your current resolution):

I was using the standard `70` CPL before but it was  causing wrap around on android 11 avd default config, so I lowered it to around `50`.

**Comment by RalfWerner** [comment]
Stupid question: Why should someone get this message when he used an _installed_ Play Store version or _does it today_?
He would get a problem only at `pkg up`!

**Comment by agnostic-apollo** [comment]
Because `deprecated` things get removed after some time, its the standard practice. It's been many months now. Moreover, new versions have lot of new features and bug fixes and users on old versions are often reporting issues in multiple repositories which we then have to deal with. Playstore page of `termux-app` has been filled with bad reviews of "broken app", even though its clearly mentioned on the page that app is not being updated, cause users often don't bother to read or search. Also asking people to pay for plugins when the app at installation time is broken and has bugs is unethical. Old versions also don't have debugging and crash reports support to actually help solve bugs. Its also easier for us to solve package related issues and provide custom functionality with app updates. The amount of bug reports we have received for that ~~damn~~ bintray issue, that we are actually still receiving reports for on multiple platforms after months, could have been drastically reduced if users had latest bootstrap installed which had working repos. Repo issues could technically also have been solved with an app update by automatically replacing faulty ones with `sed` but at the time `termux-app` didn't have such utils implemented to do stuff like that, now `termux-shared` does, so if similar issues are faced in future, they can be handled.

**Comment by RalfWerner** [comment]
Thanks for your detailed answer - I'm confirm with it!
My question was meant more _practic_! So the person _someone_ would get the message at all through this PR?

Actually, only with the knowledge of tomorrow the messages can be displayed with a version of today. Example:
Helpful would be a message: `[[ "$TERMUX_VERSION" > "0.117" ]]&& cat @TERMUX_PREFIX@/etc/remove-float-widget-apps`

File: packages/termux-tools/motd-playstore [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+so you must move to F-Droid or Github Debug
+builds. You can follow instructions at
+https://wiki.termux.com/wiki/Backing_up_Termux
+to backup all your Termux app data under $HOME/
+and $PREFIX/ and then restore it once you
+re-install the alternate builds. If done right,
+you will not lose any existing data when moving.
+Check https://github.com/termux/termux-app#installation
+for more info on how to intall other builds and
+on deprecation details. Check
```
Comment (buttaface) [inline packages/termux-tools/motd-playstore:20]: install from F-Droid or github and more deprecation details.

**Review by buttaface — COMMENTED** [review]
Some better wording below.

**Comment by agnostic-apollo** [comment]
Moved to #7493

**Pull request closed without merging**